outputs/kubernetes.go

```diff
@@ -0,0 +1,130 @@
+package outputs
```

Comment:
@simar7 
you just added `exec` output. Is there any chance you can use it and shell script that calls `kubectl annotate` ?

Reply:
we certainly can - the idea is to promote Kubernetes as a first class citizen. I am with you on the fact that anything can be done with just a shell and an http request :)

Reply:
AquaServer can be integrated with k8s itself :
https://dzone.com/articles/implementing-aqua-security-to-secure-kubernetes
so from app scopes perspective it looks like some overlap that may lead to confusion.